### ISSUE TITLE
precise-commits inserts semicolon in middle of expression

### ISSUE THREAD

**Issue by djcsdy** [softwareventures/precise-commits#45]
```txt
✔  precise-commits: 3 modified file(s) found
✔  [1/3] Processing file: xxxx-ui/xxxx-offline-data/__tests__/offline-data-test.ts
✖  precise-commits: An Error occurred
SyntaxError: ')' expected. (77:10)
  75 |             expect(firstResult).toBeNull();
  76 |             expect(commandEffectsReceived.length).toBe(0); // NOTE: .toBeEmpty() is on .d.ts but is not found - expect() is extended in mysterious ways
> 77 |         };, {timeout: 150});
     |          ^
  78 |     });
```

**Comment by djcsdy** [comment]
Turns out this was fixed in prettier sometime between v2.5.1 and v2.8.8.

**Comment by djcsdy** [comment]
Added a test verifying that this is fixed.
